definitions/npm/datauri-download_v0.1.x/flow_v0.25.x-/datauri-download_v0.1.x.js

```diff
@@ -0,0 +1,15 @@
+declare module "datauri-download" {
+  declare module.exports: (
+    filename: string,
+    type: string,
+    data: string | ArrayBuffer | Blob
+  ) => void;
+}
+
+declare module "datauri-download/index" {
+  declare module.exports: $Exports<"datauri-download">;
+}
+
+declare module "datauri-download/index.js" {
+  declare module.exports: $Exports<"datauri-download">;
+}
```

Comment:
You don't need to re-decalare `index`, `index.js` it's already covered by `module.exports`.

Reply:
Agreed. Go ahead and fix that first and we'll get this merged in. :)